plugins/filters/askAI/askAI.cm

```diff
@@ -0,0 +1,14 @@
+triggers:
+  exclude:
+    branch:
+      - r/dependabot/
+
+automations:
+  generate_pr_desc:
+    if:
+      - true
+    run:
+      - action: add-comment@v1
+        args:
+          comment: |
+            {{ code | askAI("Based on the given context, search for new functions without tests and suggest the tests to add. If all functions are covered completely, return 'no tests to suggest.'", env.OPEN_AI_TOKEN) | encode }}
```

Comment:
What's `code`? is it a new context? where is is defined?

Reply:
Same for `encode` - is it documented?

Reply:
Adding the `encode` and `decode` documentation on a separate PR